.github/workflows/cmake.yml

```diff
@@ -0,0 +1,55 @@
+name: CMake
```

Comment:
We can change the name here to reflect the actual project + context.

Reply:
done